src/main/java/bank/currencyconverter/CurrencyApiCurrencyConverter.java

```diff
@@ -0,0 +1,36 @@
+package bank.currencyconverter;
+
+import org.springframework.web.reactive.function.client.WebClient;
+
+import java.util.Currency;
+
+public class CurrencyApiCurrencyConverter implements CurrencyConverter {
+    private WebClient webClient;
```

Comment:
```suggestion
    private final WebClient webClient;
```
і ініціалізувати в конструторі